tests/test_security_curve.cpp

```diff
@@ -40,6 +40,10 @@
 #  include <unistd.h>
 #endif
 
+#include "../src/tweetnacl.h"
```

Comment:
I can imagine only few things worse than duplicating code ;)
I don't think it would break applications... libzmq can still contain the inner library (static libraries can be merged).